great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -86,6 +107,29 @@ def __init__(self):
         )
 
 
+class GXCloudConfigError(Exception):
+    def __init__(self, missing_keys: list[str]):
+        super().__init__(
+            "At least one of the following required keys are missing from the GX Cloud config: "
+            f"{missing_keys}"
+        )
+
+
+OPTIONAL_CLOUD_CONFIG_KEYS = [GXCloudEnvironmentVariable.WORKSPACE_ID]
+
+
+@dataclass
+class Workspace:
+    id: str
```

Comment:
don't have much context but wondering if str type intentional (looks like user_id below is UUID type)?